howto/container/access.md

```diff
@@ -30,8 +30,8 @@ In the next sections, you will learn how to connect a container running on Anbox
 ### Install scrcpy
 
 Scrcpy is not available from the official Ubuntu repositories. Therefore, you must install scrcpy in one of the following ways:
-
-* Build scrcpy from the source by following the official [guide](https://github.com/Genymobile/scrcpy/blob/master/BUILD.md)
+<!-- wokeignore:rule=master -->
+* Build scrcpy from the source by following the official [guide](https://github.com/Genymobile/scrcpy/blob/master/doc/build.md)
```

Comment:
Isn't it better to have a permalink instead, to avoid similar issues in the future?
```suggestion
* Build scrcpy from the source by following the official [guide](https://github.com/Genymobile/scrcpy/blob/4ad74794255b7669a89c023afc7ae7a2c0da4dc3/doc/build.md)
```

But then, this link may get outdated. It may be better to have a broken link as it may be easier to spot. What do you think?

Reply:
That's exactly what I was thinking, we can easily spot the broken link with the link checker. I think we can retain it for now and see how often it changes. If we end up fixing a broken link too often, then the permalink might be worth using.